projects/control-service/projects/pipelines_control_service/src/test/java/com/vmware/taurus/service/JobImageDeployerTest.java

```diff
@@ -155,6 +157,7 @@ public void testScheduleLabelsAnnotations() {
       Assertions.assertEquals("scheduled/runtime", startedBy);
       Assertions.assertEquals("scheduled", executionType);
       Assertions.assertEquals("false", unscheduled);
+      Assertions.assertEquals("3.9", pythonVersion);
```

Comment:
What about tests scenarios of create and update where no explicit version is set?